src/views/createEditCommunity/state.ts

```diff
@@ -446,107 +446,129 @@ export const validateField = (
     state: INITIAL_FORM_STATE,
     dispatch: React.Dispatch<FormActionTypes>
 ) => ({
-    name: () => {
-        dispatch({
-            type: formAction.SET_NAME_VALID,
-            payload: state.name.length > 0,
-        });
+    name: (v: boolean = true) => {
```

Comment:
We should try to keep variables names as clear and representative as possible.

Reply:
that's right. Honestly, I couldn't come up with a name. Any suggestion?

Reply:
Just replaced with isValidated

Reply:
Man, that's confusing. The goal of that parameter is to decide if the state should be updated or not.
Don't do this again, changing things and merging afterward.